cpp/src_prims/sparse/coo.h

```diff
@@ -445,7 +457,7 @@ __global__ void coo_row_count_nz_kernel(int *const rows, T *const vals, int nnz,
 }
 
 template <int TPB_X, typename T>
-__global__ void coo_row_count_scalar_kernel(int *const rows, T *const vals,
+__global__ void coo_row_count_scalar_kernel(const int *rows, const T *vals,
```

Comment:
Another custom kernel.

Reply:
Also a degree computation for sparse data formats. I might be misunderstanding your use of “custom kernel” here.